scripts/docker/tasking-manager/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM python:3.7-buster
+ARG PYTHON_IMG_TAG=3.10
+FROM python:${PYTHON_IMG_TAG}-bullseye
```

Comment:
Maybe in a different PR, but if we can get away with using  `python:3.10-slim-bullseye` for this, we will save ourselves a lot of security headaches later.

Reply:
We'd probably want to use `python:3.11-slim-bookworm` (Debian Bookworm is being released on June 10th), assuming all tests pass.

But I want to say someone was looking at reworking the `Dockerfile`s at some point. As a specific example, I don't think this Dockerfile is being used (`RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -`). I think the `Dockerfile.{backend|frontend}` is what is used elsewhere.

Reply:
Re: bookworm, agreed. 


Re: Dockerfile, yes, a review / rewrite is due. I think we have ignored frontend Dockerfile for a while mostly because it doesn't affect our own deployments. We deploy to AWS S3/Cloudfront currently. I'll make a new issue for it.

Reply:
Incidentally, tomorrow is the date for a feature freeze - https://lists.debian.org/debian-devel-announce/2023/04/msg00007.html